example-get-started/deploy.sh

```diff
@@ -17,7 +17,9 @@ popd
 
 # Requires AWS CLI and write access to `s3://dvc-public/code/get-started/`.
 mv $PACKAGE_DIR/$PACKAGE .
-aws s3 cp --acl public-read $PACKAGE s3://dvc-public/code/get-started/$PACKAGE
+#aws s3 cp --acl public-read $PACKAGE s3://dvc-public/code/get-started/$PACKAGE
```

Comment:
Is this intentionally left like this?

Reply:
I get it for now, just making sure we don't forget to update it.

Reply:
yep, I'll remove it today before the merge